Bic c

Farnazmdi opened the pull request:
@aarmey I realized something. These are the parameters for the 4 conditions of lapatinib, for 3 states.
The parameters for states1, 2, and 3 are different across conditions. I thought they should be the same.
I will look and see where should be changed tomorrow. 

![image](https://user-images.githubusercontent.com/34812750/107196797-ece7c800-69a7-11eb-80b2-e9d8ee13eb11.png)

aarmey commented:
Yep—looks like only G1 is set correctly.

Farnazmdi commented:
Sorry I labeled them incorrectly. Shape G2 and scale G1 have to change places.
But I meant among different conditions. Doesn't state 1 have to be the same in all conditions?

aarmey commented:
Farnaz, at this point you _need_ to be familiar with these concepts from the model. If these are different lineages in the same condition, yes, they should be identical.

Farnazmdi commented:
Population number 0 is control, population number 1 is treatment with 25 nM lapatinib, ...
 
I think the parameters of state 1 in all conditions should be the same, but they are not here, only the shape is. While the shape has to be shared among all states and all conditions. I am suggesting that we do this modification. Because right now, state 1 has a slightly different meaning in control condition than in 25 nM treatment than in other conditions.

aarmey commented:
Shape is shared across treatment conditions, the other parameters are not. If all the parameters were shared across the conditions, there would be no way for a drug to have any effect.

Farnazmdi commented:
Understood. Sorry, it was a very bad mistake. I am very tired and not thinking clearly.

Farnazmdi commented:
I know that figure 9 ylabel says AIC. Here I replaced AIC with BIC and changed the text accordingly. I will run the edited figure 9 tonight and will update it in the morning. 
Other than this, it should be ready to merge. All figures are up to date and some parts of the text are modified.

aarmey commented on lineage/figures/figure9.py:
```diff
@@ -20,59 +20,58 @@ def makeFigure():
 
-    lapAIC = find_AIC(lapatinib, desired_num_states)
-    gemAIC = find_AIC(gemcitabine, desired_num_states)
+    lapAIC = find_AIC(lapatinib, desired_num_states, num_cells=5290)
```
What is the numcells argument here for?

Farnazmdi replied:
The BIC needs the number of data points, I printed out the number of cells in both drug treatments and placed it here. I didn't change the name of the function, but it is now based on BIC.

aarmey commented on lineage/figures/figure9.py:
```diff
@@ -20,59 +20,58 @@ def makeFigure():
-    lapt_tHMMobj_list, lapt_states_list, _ = Analyze_list(lapatinib, lpt_st, fpi=True)
-    gemc_tHMMobj_list, gemc_states_list, _ = Analyze_list(gemcitabine, gmc_st, fpi=True)
+    # # assign the predicted states to each cell
+    # for idx, lapt_tHMMobj in enumerate(lapt_tHMMobj_list):
```
Is this commented code still needed?

Farnazmdi replied:
Not now because we have the pickled files, but if for any reason we need to run the analysis again, we will. Should I remove it?

aarmey reviewed (changes requested):
Just two questions.

aarmey reviewed (approved):
Ok, this is good as-is.
